index.html

```diff
@@ -9,6 +9,7 @@
 
 </style>
 	<body>
+	<h1>Hello</h1>
```

Comment:
Why did you add this code? What it does? We don't need it

Reply:
I just Liked it